http/polytracker/httpd_harness

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+PID_FILE=/usr/local/apache2/logs/httpd.pid
+
+if [ -f $PID_FILE ]; then
+  APACHE_PID=$(cat $PID_FILE)
+  if ps -p $APACHE_PID > /dev/null; then
+    echo "Apache is already running in a different process; waiting for it to end..."
+    while ps -p $APACHE_PID > /dev/null; do
+      sleep 1
+    done
+    rm -f $PID_FILE
+  fi
+fi
+
+#APACHE_ROOT=/usr/local/apache2
```

Comment:
I think it would be worth moving the apache-related files to a subdirectory for polytracker so they can't conflict with the configuration from the other apache parser.

Some of the config files for the `apache` parser are already in subdirectories (the PID files and index.php), but some (module configs, the server config, etc) are at top level and should also be moved. I can handle this if you want.